models/Question.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let Question = new Schema({
+    id: {
+        type: number
+    },    
+    
+    name: {
+        type: String
+    },
+    answer1: {
+        type: String
+    },
+    answer2: {
+        type: String
+    },
+    answer3: {
+        type: String
+    },
+    answer4: {
+        type: String
+    },
+    answer5: {
+        type: String
+    }
```

Comment:
This being the question model we do not need to have a set number of answers predefined.

Proposed Schema
QUESTION:
description : String
assessment_id : number
answership : [need to research how to setup this field] (has_many relationship)
updated_at : {type: Date, default: Date.now() }

We can discuss this further in an upcoming meeting when we go over all our schemas for the assessment.